modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp

```diff
@@ -100,6 +99,10 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 
 	<div class="message-container"></div>
 
+	<%
+	long folderId = ParamUtil.getLong(request, "folderId");
+	%>
```

Comment:
@AliciaGarciaGarcia Shouldn't we use the `folderId` we get as a tag attribute as a default value?
```
long folderId = ParamUtil.getLong(..., "folderId", folderId);
```

Reply:
Yeah, it can be
I will add this improvement on an additional commit

Thanks for pointing at it.